src/sbvr-api/errors.ts

```diff
@@ -1,5 +1,14 @@
 import TypedError = require('typed-error');
 
+export class PayloadError extends TypedError {
+	constructor(
+		name: string,
+		public payload: { [k: string]: any } | undefined
```

Comment:
NIT: `{ [k: string]: any }` is commonly referred as `AnyObject`, it might be useful make it a separate type.